src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline

```diff
@@ -3350,6 +3350,7 @@ import org.apache.commons.cli.Option;
 import org.apache.commons.cli.Options;
 import com.google.api.gax.rpc.BidiStream;
 import com.google.example.library.v1.Comment;
+import com.google.example.library.v1.Comment;
```

Comment:
This seems to be caused by an existing bug in the generator. #2777 would fix this.

Also, artman will remove the duplicate imports anyway.